src/common/libflux/event.c

```diff
@@ -201,13 +202,15 @@ static flux_future_t *wrap_event_rpc (flux_t *h,
     flux_future_t *f;
 
     if (src) {
-        size_t dstlen = sodium_base64_encoded_len (srclen,
-                                            sodium_base64_VARIANT_ORIGINAL);
-        void *dst;
-        if (!(dst = malloc (dstlen)))
+        size_t dstbuflen = base64_encoded_length (srclen) + 1; /* +1 for NUL */
+        char *dst;
+        if (!(dst = calloc (1, dstbuflen)))
+            return NULL;
```

Comment:
The buffer for `base64_encode()` does not need to be zero-initialized because
> @note dest will be nul-padded to destlen (past any required padding)

So use `malloc()` not `calloc()`?

Reply:
aww crud, @grondo and I had this discussion in #3778 and we had come to the conclusion to use `calloc`.  But I'm guessing that note you mention above was missed.  But now that I look more carefully, that note is clearly right there.  And with `base64_decode()` too.  LMK go back and look at the `lioencode` changes from #3778 as well.